src/components/side_panel/find_and_replace/find_and_replace_store.ts

```diff
@@ -17,6 +17,11 @@ enum Direction {
   next = 1,
 }
 
+interface RefreshSearchOptions {
+  doNotJumpToMatchSheet?: boolean;
+  doNotSelectNextMatch?: boolean;
```

Comment:
Personal option, but I always prefer non-negative names, to avoid `!doNotJumpToMatchSheet` (not do not jump to match sheet). IMO, it's easier to read
```
if (jumpToMatchSheet) { ... }
```
than
 ```
 if (!doNotJumpToMatchSheet) { ... }
 ```

Reply:
Yeah I tend to agree. I did that because I like truthy default arguments even less. But I can just make the arguments mandatory for the 4 calls to refreshSearch/selectNextCell, it's not a big deal